k8s/upgrade-job/src/events/event_recorder.rs

```diff
@@ -159,12 +160,39 @@ impl EventRecorderBuilder {
             resource_version: None,
         };
 
+        let (tx, mut rx) = mpsc::channel::<Event>(10);
```

Comment:
Consider using unbound here, or a much larger buffer perhaps. Otherwise the sender may block or fail, please check the behaviour of the channel type you are using.

Reply:
Done. Added unbounded channel.